pjlib/include/pj/os.h

```diff
@@ -381,6 +381,28 @@ PJ_DECL(pj_status_t) pj_thread_get_stack_info(pj_thread_t *thread,
 #  define pj_thread_get_stack_info(thread,f,l)      (*(f)="",*(l)=0)
 #endif  /* PJ_OS_HAS_CHECK_STACK */
 
+/**
+ * @}
+ */
+
+/* **************************************************************************/
+/**
+ * @defgroup PJ_ANDROID_OS Android OS Specific
+ * @ingroup PJ_OS
+ * @{
+ * Functionalities specific to Android OS.
+ *
+ */
+
+/**
+ * Set the Java Virtual Machine environment variable for JNI purposes.
+ * Note that applications typically do not need to call this function unless
+ * PJ_JNI_HAS_JNI_ONLOAD is disabled.
+ *
+ * @param jvm           The Java Virtual Machine environment.
+ */
+PJ_DECL(void) pj_android_jni_set_jvm(void *jvm);
```

Comment:
Why use `pj_android_jni` prefix instead of `pj_jni`, as basically it is generic JNI (?), so it will be in the same group as `pj_jni_attach/detach_jvm()`.